From klloyd@sierrawireless.com Fri Mar 14 00:52:33 2008
From: Kevin Lloyd <klloyd@sierrawireless.com>
Date: Fri, 14 Mar 2008 00:53:24 -0700
Subject: USB: sierra: add another device id
To: "Greg KH" <greg@kroah.com>, <linux-usb@vger.kernel.org>
Cc: "Oliver Neukum" <oliver@neukum.org>, "Hamish Moffatt" <hamish@cloud.net.au>
Message-ID: <3415E2A2AB26944B9159CDB22001004D0262E170@nestea.sierrawireless.local>


Add support for the MC8775 device to the sierra driver.

Signed-off-by: Kevin Lloyd <klloyd@sierrawireless.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/sierra.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/sierra.c
+++ b/drivers/usb/serial/sierra.c
@@ -163,6 +163,7 @@ static struct usb_device_id id_table [] 
 	{ USB_DEVICE(0x1199, 0x6803) },	/* Sierra Wireless MC8765 */
 	{ USB_DEVICE(0x1199, 0x6812) },	/* Sierra Wireless MC8775 & AC 875U */
 	{ USB_DEVICE(0x1199, 0x6813) },	/* Sierra Wireless MC8775 (Thinkpad internal) */
+	{ USB_DEVICE(0x1199, 0x6815) },	/* Sierra Wireless MC8775 */
 	{ USB_DEVICE(0x1199, 0x6820) },	/* Sierra Wireless AirCard 875 */
 	{ USB_DEVICE(0x1199, 0x6832) },	/* Sierra Wireless MC8780*/
 	{ USB_DEVICE(0x1199, 0x6833) },	/* Sierra Wireless MC8781*/
